src/components/ContextMenuItem.js

```diff
@@ -34,6 +34,12 @@ const propTypes = {
 
     /** The action accept for anonymous user or not */
     isAnonymousAction: PropTypes.bool,
+
+    /** Whether the menu item is focused or not */
+    focused: PropTypes.bool,
```

Comment:
Let's rename this to `isFocused` for consistency, since it's a boolean

```suggestion
    isFocused: PropTypes.bool,
```